pandas/io/parsers.py

```diff
@@ -716,7 +722,8 @@ def parser_f(filepath_or_buffer,
 
 
 @Appender(_read_fwf_doc)
-def read_fwf(filepath_or_buffer, colspecs='infer', widths=None, **kwds):
+def read_fwf(filepath_or_buffer, colspecs='infer', widths=None,
+             infer_nrows=None, **kwds):
```

Comment:
shouldn't this be 100?

Reply:
I might be bending over to maintain backwards compatibility here (or I might be wrong), but I decided to let `infer_nrows` default to `None` so that if there is old code that had explicitly called `detect_colspecs(n=1234)`, the function would behave as they expected. If someone sets `infer_nrows` to a number, then `n` is overwritten with `infer_nrows`.

Reply:
To clarify a little further, this was the only way I could think to respect old code that explicitly set `n` in a direct call to `detect_colspecs`. Any new code that chooses to use `infer_nrows` will overwrite `n`, but I did not want to automatically overwrite `n` with a default of 100 if someone had already chosen to do otherwise.

Reply:
this is still backward compat. If n is not None, use it, else use infer_nrows. (and let's deprecate setting n that way anyhow).